tests/vagrant/vagrant-template-centos6/roles/install-pkgs/tasks/main.yml

```diff
@@ -77,6 +77,7 @@
     - which
     - xfsprogs
     - yajl-devel
+    - xxhash
```

Comment:
Isn't this xxhash-libs ? (not that I think centos6 is relevant!)

Reply:
No it's the binary which has to be installed. It is just for the .t tests.